Project.toml

```
@@ -0,0 +1,23 @@
+name = "FFTW"
+uuid = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+version = "0.3.0-DEV"
```

The -DEV part isn't needed
```suggestion
version = "0.3.0"
```

---

I was under the impression that only the release commit should have a non-dev version, apparently I misunderstood this.

---

Nope. When checking something out to a particular branch (including master), Pkg just recognizes the version as `x.y.z+`, where `x.y.z` is what's declared in the Project.toml `version`. The only time `version` actually matters is for registering and for compatibility bounds.